packages/frontend/src/components/MkPostForm.vue

```diff
@@ -896,6 +903,11 @@ async function post(ev?: MouseEvent) {
 			if (m === 0 && s === 0) {
 				claimAchievement('postedAt0min0sec');
 			}
+			if (props.initialNote) {
```

Comment:
```suggestion
			if (props.initialNote && props.deleteInitialNoteAfterPost) {
```

こちらの条件が反映されていないようなきがします

Reply:
たしかに。。。家戻ったら対応します。。。

Reply:
何度もお手を煩わせてしまいすみません。。。
今度こそ修正しました！

加えて、実績獲得の式がイマイチだったことに気づいたので修正しています(date.getTime()してなかった)。
> date - new Date(props.initialNote.createdAt).getTime() < 〜省略〜
> date.getTime() - new Date(props.initialNote.createdAt).getTime() < 〜省略〜